README.md

```diff
@@ -2,9 +2,11 @@
 
 [![PyPI version](https://badge.fury.io/py/vertica-python.png)](http://badge.fury.io/py/vertica-python)
 
-0.4.2 breaks some of the older query interfaces (row_handler callback, and connection.query).
+0.4.x breaks some of the older query interfaces (row_handler callback, and connection.query).
 It replaces the row_handler callback with an iterate() method. Please see examples below
 
+If you are on 0.4.1 - 0.4.3, please upgrade to 0.4.5 as there are various bug fixes
```

Comment:
we went to 0.4 because of the backwards incompatibility. 0.4.5 is just fixing commit/transaction regressions.